packages/react-native-editor/__device-tests__/helpers/utils.js

```diff
@@ -450,26 +450,16 @@ const swipeFromTo = async (
 	to = defaultCoordinates,
 	delay
 ) =>
-	driver.touchPerform( [
-		{
-			action: 'press',
-			options: from,
-		},
-		{
-			action: 'wait',
-			options: {
-				ms: delay,
-			},
-		},
-		{
-			action: 'moveTo',
-			options: to,
-		},
-		{
-			action: 'release',
-			options: {},
-		},
-	] );
+	await driver
+		.action( 'pointer', {
+			parameters: { pointerType: 'touch' },
+		} )
+		.move( { ...from, duration: 0 } )
+		.down( { button: 0 } )
+		.move( { ...to, duration: 300 } )
+		.up( { button: 0 } )
+		.pause( delay )
+		.perform();
```

Comment:
As outlined in https://github.com/WordPress/gutenberg/pull/55322/commits/bf527eec76635a8a6eaf1fd2e441ed836487acdc, this refactors to avoid the deprecated `touchPerform` command. It was done to address recent [CI task failures](https://github.com/WordPress/gutenberg/pull/55322#issuecomment-1764742815). It is unclear why the usage of this command caused seemingly sporadic test failures, but it is likely good to avoid this regardless. 

@geriux thank you for the [note on log levels](https://github.com/WordPress/gutenberg/pull/55322#issuecomment-1764767359). It allowed me to observe the failure occurring immediately after this `touchPerform` call. 🙇🏻

Reply:
Glad it helped! And TIL about the deprecated command! Thanks for making the changes 🚀